test/shell_runner.go

```diff
@@ -21,15 +21,18 @@ type ShellRunner struct {
 	// dir contains the directory path in which this runner runs.
 	dir string
```

Comment:
Can we rename this to localDir / cwd since we now have globalDir? Also thoughts on globalDir being renamed to homeDir?

Reply:
Good suggestion. I went with `workingDir` and `homeDir`. Didn't use `cwd` as it has a different meaning, the currently active directory in the shell.